var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -120,6 +121,12 @@ class Cp2k(MakefilePackage, CudaPackage, CMakePackage, ROCmPackage):
             when="@:7",  # req in CP2K v8+
             description=("Use CUBLAS for general matrix operations in DBCSR"),
         )
+        variant(
+            "cusolvermp",
+            default=True,
+            when="@2023.2:",
+            description=("Use cuda eigensolver with GPU support"),
```

Comment:
```suggestion
            description="Use Nvidia cuSOLVERMp eigensolver",
```
Description is a bit too generic.